forge-gui/res/cardsfolder/upcoming/the_red_terror.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Red Terror
+ManaCost:3 R
+Types:Legendary Creature Tyranid
+PT:4/3
+T:Mode$ DamageDone | ValidSource$ Card.RedSource+YouCtrl | ValidTarget$ Player,Permanent | Execute$ TrigPutCounter | TriggerDescription$ Advanced Species — Whenever a red source you control deals damage to one or more permanents and/or players, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+DeckHints:Color$Red
```

Comment:
Not needed... since this is a red card, stands to reason the deck will have red in it

Reply:
@Northmoc Was thinking it is better in mono red but I guess there's no hint for that